vector/src/main/res/layout/fragment_room_detail.xml

```diff
@@ -108,15 +108,31 @@
         app:layout_constraintTop_toBottomOf="@id/syncStateView"
         tools:listitem="@layout/item_timeline_event_base" />
 
-    <im.vector.riotx.core.ui.views.JumpToReadMarkerView
-        android:id="@+id/jumpToReadMarkerView"
+
+    <FrameLayout
```

Comment:
This should be a Vertical LinearLayout, to be able to display both children

Reply:
Nope, design by Rok want the jump to be on top of the widget

Reply:
🤮